python/tvm/relay/frontend/onnx.py

```diff
@@ -923,6 +924,18 @@ def _impl_v1(cls, inputs, attr, params):
         return _op.erf(inputs[0])
 
 
+class ConstantOfShape(Elemwise):
+    """Operator converter for ConstantOfShape
+    """
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        if not isinstance(inputs, list) or len(inputs) < 2:
+            raise ValueError("Expect minimum 2 inputs")
+        # reps: The number of times repeating the tensor data.
+        shape = tuple(params[inputs[1].name_hint].asnumpy().astype('int').tolist())
```

Comment:
Will [full](https://docs.tvm.ai/api/python/relay/op.html?highlight=full#tvm.relay.op.transform.full) work better here?

Also, this will throw an exception if the shape is not a constant. Can you please put this section under a try block so we can display a correct error message if an exception is thrown?

Reply:
Hi @soiferj 

I think the `tile` operator will work well and  I'm open to it if you prefer to use `full`.
Also, I add the try block so we can catch the error if an exception is thrown.